lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -1085,7 +1085,9 @@ class SPIRVSwitch : public SPIRVInstruction {
     return static_cast<SPIRVBasicBlock *>(getValue(Default));
   }
   size_t getLiteralsCount() const {
```

Comment:
The function name is unfortunate. It doesn't return number of literals, but it returns number of words required for each literal. @svenvh could you rename it to something like `getLiteralSize` ?
Other than that LGTM. Thanks!

Reply:
Thanks!  Renamed to `getLiteralSize`